ui/app/templates/workspace/projects/project/app.hbs

```diff
@@ -9,7 +9,20 @@
 )}}
 <PageHeader @iconName="git-repository">
   <div class="title">
-    <h1>{{@model.application.application}}</h1>
+    <div class="title-wrapper">
```

Comment:
Not sure I entirely needed to introducing a wrapping element here, but I didn’t want to throw off the `title` styling.